CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.4.1]
+* Fixes an issue where if the app was displayed in full screen mode, an opaque empty toolbar would appear at the top [#249](https://github.com/GroovinChip/macos_ui/issues/249) 
```

Comment:
I know in the code we're creating an `NSToolbar`, but shouldn't it be called a titlebar? From what I know of cocoa/appkit applications, that part of the window is referred to as a titlebar. I could be wrong, though.

Reply:
<img width="793" alt="Screenshot 2022-05-14 at 11 50 45 PM" src="https://user-images.githubusercontent.com/9117427/168447737-d540dc0d-41fc-46df-857b-f3cb6335d2cd.png">

As I understand, the title bar is the thin bar that holds the apple icon and the menu. In our issue, the title bar would still appear when you hovered the mouse at the top of the full screen window (and still does). However, the empty toolbar from `NSToolbar` appeared right below it, obscuring the top part of the app's window.

Reply:
Alright, then.